pyjl/plugins.py

```diff
@@ -89,16 +89,12 @@ def visit_ap_dataclass(self, cls):
         return cls
 
     def visit_range(self, node, vargs: List[str]) -> str:
+        assert 0 < len(node.args) < 4, "range() call with unexpected arguments"
```

Comment:
Do you want me to remove the `assert` or not?

Reply:
My suggestion is to keep it, convert the assert to a `raise AppropriateException` and then implement a flag in `clike._dispatch()` to catch an entire class of exceptions and let the transpilation proceed.

This way: `range(1,2,3,4)` would get transpiled into the target language under the flag (fails later in the pipeline), but allows us to discover more problems in the source. The flag itself can be another PR if you prefer.